editor.planx.uk/src/pages/Preview/StatusPage.tsx

```diff
@@ -57,7 +50,6 @@ const StatusPage: React.FC<Props> = ({
   });
 
   const theme = useTheme();
```

Comment:
question: as we're moving away from `useStyles()` should we also be aiming to move away from `useTheme()` in favor of `sx` with the `theme` callback value?

Reply:
This is a good question!

`useTheme()` is not deprecated in MUI v5 like `useStyles()` is, so we can safely continue using it. There are some cases where we likely can't avoid it just now - for example in this component we're using `theme` to pass a value into another component as a custom prop (so `sx` with a callback wouldn't work).

If there are places we're using `useTheme()` just to pass a value where we we could use `sx`, I'd probably favour `sx` as being a bit neater and and terser, but I think it's really just a personal preference.